Content.Shared/Hands/EntitySystems/SharedHandsSystem.Drop.cs

```diff
@@ -129,6 +130,7 @@ public bool TryDrop(EntityUid uid, Hand hand, EntityCoordinates? targetDropLocat
         if (targetDropLocation == null || isInContainer)
         {
             TransformSystem.DropNextTo((entity, itemXform), (uid, userXform));
+            RaiseLocalEvent(entity, new DroppedEvent(uid), true);
             return true;
```

Comment:
I'm kind of suss that this bypasses droppedinteraction + dodrop entirely as it foregoes admin logging, inputmover etc and feel it should just be calling those methods.